### ISSUE TITLE
Please post release notes or create CHANGELOG.md for vega-embed updates

### ISSUE THREAD

**Issue by RandomFractals** [vega/vega-embed#368]
so that it's easy to follow vega-embed library changes and reference them in other places via release tags or links to changelog sections.

I really like how you do it consistently for vega-lite releases: 

https://github.com/vega/vega-lite/releases

This would make it easier for me to grab your latest and integrate it into VegaViewer vscode extension since I try to keep it up to date for devs using that IDE & vega/vega-lite.

& thanks for your hard work on the whole vega vis ecosystem!

**Comment by domoritz** [comment]
We need to do this for all Vega related libraries and it would be best to automate it as much as possible.

**Comment by domoritz** [comment]
For now, you can take a look at the changelog produced by github such as https://github.com/vega/vega-embed/compare/v6.5.0...v6.5.1. 

<img width="665" alt="Screen Shot 2020-03-21 at 08 44 30" src="https://user-images.githubusercontent.com/589034/77230246-304b0500-6b50-11ea-93b5-93fbd0604b53.png">

**Comment by RandomFractals** [comment]
yeah, I did. my request was for future releases. Thanks!

**Comment by RandomFractals** [comment]
@domoritz could you please post short release notes in your vega-embed releases or create changelog.md It's still hard to follow the changes made to this library with all the bot dependencies updates you are using for commits. Thanks in advance!

**Comment by domoritz** [comment]
Check out the release notes now.
